git_automation.py

```
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+import subprocess
+import os
+import sys
+
+def run_git_command(command, description):
+    """Execute a git command and handle errors"""
+    print(f"\n=== {description} ===")
+    try:
+        result = subprocess.run(command, shell=True, cwd='/home/vibecoder/neonpro', 
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)